recipes/aws-iam-authenticator/meta.yaml

```diff
@@ -0,0 +1,56 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 13 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "aws-iam-authenticator" %}
+{% set version = "1.11.5" %}
+{% set version_date = "2018-12-06" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://amazon-eks.s3-us-west-2.amazonaws.com/{{ version }}/{{ version_date }}/bin/linux/amd64/aws-iam-authenticator  # [linux]
+  sha256: a46c66eb14ad08204f2f588b32dc50b10e9a8a0cc48ddf0966596d3c07abe059  # [linux]
+
+  url: https://amazon-eks.s3-us-west-2.amazonaws.com/{{ version }}/{{ version_date }}/bin/darwin/amd64/aws-iam-authenticator  # [osx]
+  sha256: ef60d7ea5108b4de19f7fe59514643a7665df6832f94be620405138d8f60dd18  # [osx]
+
+  url: https://amazon-eks.s3-us-west-2.amazonaws.com/{{ version }}/{{ version_date }}/bin/windows/amd64/aws-iam-authenticator.exe  # [win]
+  sha256: d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f  # [win]
+
+build:
+  number: 0
+
+test:
+  commands:
+    - aws-iam-authenticator -h  # [unix]
+    - aws-iam-authenticator.exe -h  # [win]
+
+about:
+  home: https://github.com/kubernetes-sigs/aws-iam-authenticator
+  license: Apache License 2.0
```

Comment:
The linter is complain because of the word ``License`` here
It should be ``Apache 2.0``

Reply:
I mean
```
license: Apache 2.0
```

Reply:
Yes, that's it ;)